src/Security/SecurityConfigUpdater.php

```diff
@@ -102,23 +111,55 @@ public function updateForAuthenticator(string $yamlSource, string $firewallName,
             $firewall['entry_point'] = $authenticatorClass;
         }
 
+        $newData['security']['firewalls'][$firewallName] = $firewall;
+
         if (!isset($firewall['logout']) && $logoutSetup) {
-            $firewall['logout'] = ['path' => 'app_logout'];
-            $firewall['logout'][] = $this->manipulator->createCommentLine(
-                ' where to redirect after logout'
-            );
-            $firewall['logout'][] = $this->manipulator->createCommentLine(
-                ' target: app_any_route'
-            );
-        }
+            $this->configureLogout($newData, $firewallName);
 
-        $newData['security']['firewalls'][$firewallName] = $firewall;
+            return $this->manipulator->getContents();
+        }
 
         $this->manipulator->setData($newData);
 
         return $this->manipulator->getContents();
     }
 
+    public function updateForLogout(string $yamlSource, string $firewallName): string
+    {
+        $this->createYamlSourceManipulator($yamlSource);
+
+        $this->configureLogout($this->manipulator->getData(), $firewallName);
+
+        return $this->manipulator->getContents();
+    }
+
+    /**
+     * @legacy This can be removed once we deprecate/remove `make:auth`
+     */
```

Comment:
I think that's not technically true. It's just that once we remove `make:auth`, this won't NEED to be in a private method anymore. I'd remove this.

Reply:
The private method shouldn't be needed anymore here. We have a public `updateForLogout()` that this logic will be moved to.